### PR TITLE
chore: packaging and development improvements

### PR DESCRIPTION
**Pull request by pi0**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

- [x] Eliminate the build step that temporary required for #4051
- [x] Refactors on builder/webpack to relax dependencies and fix problems with ESM
- [x] Eliminate 'lodash/*' import warns
- [x] Use const instead of var for dist

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: scripts/dev [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import consola from 'consola'
+
+import Package from './package.js'
+
+async function main() {
+  // Read package at current directory
+  const rootPackage = new Package()
+  const workspacePackages = rootPackage.getWorkspacePackages()
+
+  // Create a dev-only entypoint to the src
```
Comment (manniL) [inline scripts/dev]: Small typo there 🙊 
```suggestion
  // Create a dev-only entrypoint to the src
```

File: packages/webpack/src/index.js [hunk 2]
```diff
@@ -1 +1 @@
-export { default } from './builder'
+export { WebpackBuilder } from './builder'
```
Comment (clarkdo) [inline packages/webpack/src/index.js]: Should be `as BundleBuilder` ?

Comment (pi0) [reply packages/webpack/src/index.js]: @clarkdo No different. Explicit names help reducing namespace conflicts (Like importing two different builders). Will change it :)

Comment (clarkdo) [reply packages/webpack/src/index.js]: I mean https://github.com/nuxt/nuxt.js/pull/4193/files#diff-4f235fc89b94513b32c2cc8b5896cc5aR92 is using `BundleBuilder`, but export name is `WebpackBuilder` 😄

Comment (pi0) [reply packages/webpack/src/index.js]: You are right! Good catch :D

File: packages/cli/src/command.js [hunk 3]
```diff
@@ -79,20 +79,25 @@ export default class NuxtCommand {
...
+
   async getNuxt(options) {
     const { Nuxt } = await this.importCore()
     return new Nuxt(options)
   }
 
   async getBuilder(nuxt) {
     const { Builder } = await this.importBuilder()
-    return new Builder(nuxt)
+    const { BundleBuilder } = await this.importWebpack()
```
Comment (clarkdo) [inline packages/cli/src/command.js]: Good way to separate the concerns !

**Review by clarkdo — APPROVED** [review]

**Pull request merged**